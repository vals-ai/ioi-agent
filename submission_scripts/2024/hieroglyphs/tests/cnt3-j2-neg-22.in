version https://git-lfs.github.com/spec/v1
oid sha256:c3c395dd9f2b0e1158715f6490c4301dac8e674ea30d5891a1e9fb16699be82f
size 1160283
