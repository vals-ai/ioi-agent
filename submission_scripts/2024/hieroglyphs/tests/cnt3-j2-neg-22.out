version https://git-lfs.github.com/spec/v1
oid sha256:5af72cb432809aeb889734ab21db1ff53e14bb02c1f5ccb54527d683323d28d5
size 386788
